debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorIT.java

```diff
@@ -2424,4 +2424,61 @@ public void testNoEmptySchemaLogWarningWithSnapshotNever() throws Exception {
 
         stopConnector(value -> assertThat(logInterceptor.containsWarnMessage(DatabaseSchema.NO_CAPTURED_DATA_COLLECTIONS_WARNING)).isFalse());
     }
+
+    @Test
+    @FixFor("DBZ-3949")
+    public void testDMLInChangeEvents() throws Exception {
```

Comment:
```suggestion
    public void testDmlInChangeEvents() throws Exception {
```

(It's the convention we adhere to for abbeviations in identifiers; I know it's not applied consistently yet in existing code, but let's try and do it right in new/changed code).